scripts/pm/hf_from_gcs.py

```diff
@@ -0,0 +1,224 @@
+from google.cloud import storage
+import os
+import re
+import subprocess
+import tempfile
+import shutil
+from collections import defaultdict
+import argparse
+
+# List of GCS directories to check
+dirs = [
+    "gs://marin-eu-west4/checkpoints/llama-8b-tootsie-0.001-19ad63/hf/",
```

Comment:
Aside: naming scheme is a bit confusing...the prefix in some sense should all be llama or tootsie...(too late now)

Reply:
Actually, we should define the HF name for the model (which might not be the internal name, following https://docs.google.com/document/d/1I1hhcxlwl34_pgrXPAm6bR8XSCMn_qgWhfbagahbbSY/edit?tab=t.0).

Reply:
For the actually uploaded Tootsie model, it'll all get pushed to marin-8b-base on HF. The different versions will get pushed as commits to the history of marin-8b-base with a reference to the original GCS path.